### DISCUSSION TITLE
Sunbird inQuiry Working Group initialization!

### DISCUSSION THREAD

**Discussion by PoojaJManjooran** [Sunbird-inQuiry/Community#77] [Announcements]
Hi folks! 

We are glad to inform that we are forming a working group for the Sunbird inQuiry building block. 
Those of you who are actively contributing to this building block or using it can nominate themselves to be a part of this group by replying to this thread. We request that there should not be more than three people from the same organization nominated to be part of this working group. The individuals who are nominated should have made significant contributions or used the building block and are SMEs on the building block.

Responsibilities of Working Group members:

1. Engage actively in the discussion forums and with the community by providing feedback to new ideas, technical or process related clarifications, helping plan events, etc.
2. Accept/Reject New Requirements, Tech Design, PRs
3. Can be sponsors for New features
4. Define community practices
5. Enable ease of adoption
6. Define Building Block roadmap
7. Advocate evolution of capabilities that increase the value-offering of the building block. 
8. Foster community growth and facilitate vibrant community.

The Working Group meets once every two weeks to connect and discuss - new ideas, open issues and take any decisions required or define action items.

The first Working Group meeting is scheduled (tentatively) on 10th May, Wednesday. The meeting link and timings will be shared here soon.

@kameshbhr @sajeshkayyath @krgauraw @rajnishdargan @vaibhavbhuva  @Naveenkumar739 @Krishnaj20 @madhucr @alok-os @SanthoshVasabhaktula @vrayulu @vinukumar-vs @pallakartheekreddy @anandp504 @reshmi-nair @surendrasinghs @maheshkumargangula @pkranga @mathewjpallan @coolbung @mahesh-maney  @sukhpreetsamagra @ravishtrigyn @anandvarada @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @kiranharidas187
@sanyapawah @aayushkedia95 @ChakshuGautam @abhay-samagra @Prateek-slokam @aks30 @devendra-tarento @sharathkashyap @snehal0904 @abhishekpnt @rajeevsathish @manojSRawat @santhosh-tarento @techframewirk @Mayank-sa

**Comment by madhucr** [comment]
Definitely the immediate need.  Thanks @PoojaJManjooran @sajeshkayyath for setting up this process and involving
 the community actively.

**Comment by vijiurs** [comment]
Dear Pooja,

Please include Prateek Agarwal ( @Prateek-slokam
<https://github.com/Prateek-slokam>),  Akash Shah ***@***.***
<https://github.com/aks30> ) and Kiran Haridas ***@***.***
<https://github.com/kiranharidas187>)  from Shikshalokam to the working
group.

Regards
Vijayashree

**Reply by PoojaJManjooran** [reply]
Sure!

**Comment by sharathkashyap** [comment]
Hello Pooja,
          Please include Sharath Prasad ([@sharathkashyap](https://github.com/sharathkashyap))
  and Santhosh Kumar from Tarento as part of working group.
  
  Regards,
  Sharath Prasad

**Reply by sharathkashyap** [reply]
Please include Devendra (https://github.com/devendra-tarento) as third member from Tarento

**Comment by PoojaJManjooran** [comment]
### Working Group Catchup!
We are having the first Working Group Catchup on **22nd May, Monday** at 5pm

**Join Zoom Meeting**
https://ekstep-org.zoom.us/j/84979107495?pwd=ZDQ5NUh0TEh2R0ZaUXp6MVlKL2xHUT09
Meeting ID: 849 7910 7495
Passcode: 892153

@kameshbhr @sajeshkayyath @krgauraw @rajnishdargan @vaibhavbhuva @Naveenkumar739 @Krishnaj20 @madhucr @alok-os @SanthoshVasabhaktula @vrayulu @vinukumar-vs @pallakartheekreddy @anandp504 @reshmi-nair @surendrasinghs @maheshkumargangula @pkranga @mathewjpallan @coolbung @mahesh-maney @sukhpreetsamagra @ravishtrigyn @anandvarada @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @kiranharidas187
@sanyapawah @aayushkedia95 @ChakshuGautam @abhay-samagra @Prateek-slokam @aks30 @devendra-tarento @sharathkashyap @snehal0904 @abhishekpnt @rajeevsathish @manojSRawat @santhosh-tarento @techframewirk @mayank-sa @sharathkashyap

**Reply by PoojaJManjooran** [reply]
Hey folks!
As planned, we will be having the Working Group Catchup today at 5pm.
In today's call we shall have the introduction of the inQuiry Working group members and also discuss the purpose and responsibility of the working group.

@kameshbhr @sajeshkayyath @krgauraw @rajnishdargan @vaibhavbhuva @Naveenkumar739 @Krishnaj20 @madhucr @alok-os @SanthoshVasabhaktula @vrayulu @vinukumar-vs @pallakartheekreddy @anandp504 @reshmi-nair @surendrasinghs @maheshkumargangula @pkranga @mathewjpallan @coolbung @mahesh-maney @sukhpreetsamagra @ravishtrigyn @anandvarada @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @kiranharidas187
@sanyapawah @aayushkedia95 @ChakshuGautam @abhay-samagra @Prateek-slokam @aks30 @devendra-tarento @sharathkashyap @snehal0904 @abhishekpnt @rajeevsathish @manojSRawat @santhosh-tarento @techframewirk @mayank-sa @sharathkashyap
